index.bs

```diff
@@ -443,8 +443,12 @@ When an {{XRSession}} is shut down the following steps are run:
   1. Set |session|'s [=ended=] value to <code>true</code>.
   1. If the [=active immersive session=] is equal to |session|, set the [=active immersive session=] to <code>null</code>.
   1. Remove |session| from the [=list of inline sessions=].
-  1. [=Reject=] any outstanding promises returned by |session| with an {{InvalidStateError}}.
-  1. If no other features of the user agent are actively using them, perform the necessary platform-specific steps to shut down the device's tracking and rendering capabilities.
+  1. [=Reject=] any outstanding promises returned by |session| with an {{InvalidStateError}}, except for any promises returned by {{XRSession/end()}}.
+  1. If no other features of the user agent are actively using them, perform the necessary platform-specific steps to shut down the device's tracking and rendering capabilities. This MUST include:
+    - Releasing [=exclusive access=] to the [=/XR device=].
+    - Deallocating any framebuffers attached by the |session| to the [=/XR device=].
```

Comment:
"Attached" doesn't seem like the right word here? Also it may not always be framebuffers specifically, so maybe something more like this (keeping in line with your recent swap chain PR)?

> Deallocating any graphics resources acquired by the |session| for presentation to the [=/XR device=].

Reply:
That makes sense. Getting on a flight soon, will push this fix later.

Reply:
Safe travels!

Reply:
Done!